app/main.py

```diff
@@ -1,6 +1,30 @@
+import requests
+import os
+
+from dotenv import load_dotenv
+
+
+load_dotenv()
+
+
+API_KEY = os.environ.get("API_KEY")
+URL = "http://api.weatherapi.com/v1/current.json?"
+FILTERING = "Kyiv"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    response = requests.get(URL, params={"key": API_KEY, "q": FILTERING})
+    if response.status_code == 200:
+        data = response.json()
+        location_name = data["location"]["name"]
+        location_country = data["location"]["country"]
+        localtime = data["location"]["localtime"]
+        temperature = data["current"]["temp_c"]
+        weather = data["current"]["condition"]["text"]
+        print(
+            f"{location_name}/{location_country} {localtime} "
+            f"Weather: {temperature} Celsiush {weather}"
+        )
```

Comment:
what if the status is not equal to 200? how to understand that something went wrong? 

add some message for this case